mozdownload/scraper.py

```diff
@@ -1008,6 +1121,14 @@ def cli():
                      help="Download a debug build")
     parser.add_option_group(group)
 
+    # Option group for try builds
+    group = OptionGroup(parser, "Try builds",
+                        "Extra options for try builds.")
+    group.add_option('--changeset',
+                     dest='changeset',
+                     help="Changeset of the try build to download")
```

Comment:
Lets try to stick with single quote signs in all the cases here. We might have to do a clean-up for older code in the future.